src/test/java/kitchenpos/application/MenuGroupServiceTest.java

```diff
@@ -0,0 +1,61 @@
+package kitchenpos.application;
+
+import kitchenpos.dao.MenuGroupDao;
+import kitchenpos.domain.MenuGroup;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.internal.verification.VerificationModeFactory;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class MenuGroupServiceTest {
+
+  @Mock
+  private MenuGroupDao menuGroupDao;
+
+  @DisplayName("메뉴그룹의 이름을 입력받아 저장할 수 있다.")
+  @Test
+  void createTest() {
+    //given
+    MenuGroup menuGroup = new MenuGroup("그룹1");
+    when(menuGroupDao.save(any())).thenReturn(new MenuGroup(1L, "그룹1"));
+    MenuGroupService menuGroupService = new MenuGroupService(menuGroupDao);
+
+    //when
+    MenuGroup savedMenuGroup = menuGroupService.create(menuGroup);
+
+    //then
+    assertAll(() -> assertThat(savedMenuGroup.getId()).isNotNull(),
+        () -> assertThat(savedMenuGroup.getName()).isEqualTo(menuGroup.getName()));
+    verify(menuGroupDao, VerificationModeFactory.times(1)).save(menuGroup);
+  }
+
+  @DisplayName("상품 목록을 반환한다.")
```

Comment:
다른 부분들은 domain별 요구사항 정리에 작성해주신 것과 매칭 시켜주셨는데요!
이 부분도 동일하게 매칭해주시면 좋을 것 같아요! 😄 
```suggestion
  @DisplayName("전체 메뉴그룹의 목록을 조회할 수 있다.")
```